sendReminders.js

```diff
@@ -69,8 +68,15 @@ var updateReminderStatus = function(reminder) {
  *
  * @return {Promise} Promise to send messages and update statuses.
  */
-module.exports.sendReminders = function() {
-  return module.exports.findReminders()
-      .then(resultArray => Promise.all(resultArray.map(r => sendReminder(r))))
-      .then(resultArray => Promise.all(resultArray.map(r => updateReminderStatus(r))))
+function sendReminders() {
+  return findReminders()
+    .then(resultArray => Promise.all(resultArray.map(r => sendReminder(r))))
+    .then(resultArray => Promise.all(resultArray.map(r => updateReminderStatus(r))));
```

Comment:
@btrott18 I was wondering why you changed the upstream code in this routine from a forEach(send then update) to a (sendall then updateall) as shown here?  

I'd think we'd want to update the record right after sending the text so we'd never end up with possibly sending a bunch of texts, experiencing a failure, and not being able to update them, and then on the next run potentially send duplicate texts...